test/integration/framework/controlplane_utils.go

```diff
@@ -62,7 +62,7 @@ import (
 // Config is a struct of configuration directives for NewMasterComponents.
 type Config struct {
 	// If nil, a default is used, partially filled configs will not get populated.
-	MasterConfig            *controlplane.Config
```

Comment:
This seems to come from #94407

@lavalamp what's your take on "control plane" vs "apiserver"? Since the package doesn't seem to include kcm or scheduler, I think control plane is misleading.

Reply:
The word to use is "Instance"

Reply:
So, InstanceConfig *controlplane.Config

Reply:
- masterConfig -> instanceConfig (type: *controlplane.Config)
- MasterReceiver -> ApiserverReceiver
- MasterHolder -> ApiserverHolder
- startMasterOrDie -> startApiserverOrDie
- NewMasterConfigWithOptions -> NewControlPlaneConfigWithOptions (include etcd) : maybe should change to Instance as well.

Updated like above